extensions/BMO/template/en/default/account/create.html.tmpl

```diff
@@ -110,52 +119,68 @@ function onSubmit() {
   <h2 class="column-header">I want to help</h2>
 
   <div id="right-blurb">
-  <p>
-  Great!  There are three things to know and do:
-  </p>
-  <ol>
-  <li>
-    Please consider reading our
-    <a href="https://developer.mozilla.org/en/Bug_writing_guidelines" target="_blank">[% terms.bug %]-writing guidelines</a>.
-  </li>
-  <li>
-    [% terms.Bugzilla %] is a public place.  Your comments and other activities
-    on [% terms.bugs %] will generally be publicly visible, and your email address will be
-    accessible through public APIs and will be visible to all logged-in users
-    of [% terms.Bugzilla %]. Some people use an
-    <a href="http://email.about.com/od/freeemailreviews/tp/free_email.htm"
-    target="_blank">alternative email address</a> for this reason.
-    See Mozilla's <a href="https://www.mozilla.org/en-US/privacy/websites/"
-    target="_blank">Websites, Communications & Cookies Privacy Notice</a> for
-    more information on our privacy policies.
-  </li>
-  <li>
-    When using [% terms.Bugzilla %] to submit patches, comments, code, and any other content,
-    you agree to our policies on open source licensing and content submission.  See Mozilla’s
-    <a href="https://www.mozilla.org/en-US/about/legal/terms/mozilla/" target="_blank">Websites
-    and Communications Terms of Use</a> for more information.
-  </li>
-  <li>
-    Please give us an email address you want to use.  Once we confirm that it works,
-    you'll be asked to set a password and then you can start filing [% terms.bugs %] and helping fix them.
-  </li>
-  </ol>
+    <p>
+      Great!  There are four things to know and do:
+    </p>
+    <ol>
+    <li>
+      Please read our
+      <a href="https://developer.mozilla.org/en/Bug_writing_guidelines" target="_blank">[% terms.bug %]-writing guidelines</a>.
```

Comment:
`[% terms.Bug %] Writing Guidelines` for consistency (end of the PR uses that form).